### PR TITLE
Update Filter and Clear

### LINKED ISSUE #241: [PE-D][Tester E] The transactions displayed after filter doesn't tell me who the transaction is from
The transactions are shown but I can't tell who those transactions originated from. This can be a little bit messy for the user.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: zbz-lvlv/ped#8

### LINKED ISSUE #252: [PE-D][Tester C] Lack of warning for `clear` functionality
Considering just how drastic the results of the `clear` command (erases essentially all data from the tracker), I think it would be nice if there is a warning to check if the user really wants to clear all entries upon execution of the command. Otherwise it might be too easy to accidentally wipe out all the data in the tracker.

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: doimoiboi/ped#4

### PR DESCRIPTION
**Pull request by jicsontoh**
-Update Filter output message and explain the output clearer in the UG
-Update Clear in UG to show warning

### CODE REVIEW

**Review by notle1706 — APPROVED** [review]
Great changes!

File: src/main/java/seedu/address/logic/commands/FilterTransCommand.java [hunk 1]
```diff
@@ -16,7 +16,8 @@ public class FilterTransCommand extends Command {
             + "Parameters: TYPE (Must be either 'buy' or 'sell')\n"
             + "Example: " + COMMAND_WORD + " buy";
 
-    public static final String MESSAGE_SUCCESS = "Filtered all %1$s transactions from all clients.";
+    public static final String MESSAGE_SUCCESS = "All %1$s transactions are filtered in the transaction panel.\n"
+            + "All clients are listed in the client panel.";
```
Comment (quannam0124) [inline src/main/java/seedu/address/logic/commands/FilterTransCommand.java:20]: I really like how clear and concise the message is!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -339,8 +339,8 @@ Example:
 
 ### Filtering buy or sell transactions: `filter`
 
-Filters the buy or sell transactions of all the clients. The filtered transactions will be displayed in the transaction panel while the client list panel will display
-all the clients.
+Filters the buy or sell transactions of all the clients. After executing this command, the filtered transactions will be 
+displayed in the transaction panel while the client list panel will display all the clients.
```
Comment (quannam0124) [inline docs/UserGuide.md:343]: This explanation corresponds with the success message in your command. Good job!

**Review by quannam0124 — APPROVED** [review]
LGTM! Good job Jicson!

**Review by ElginL — APPROVED** [review]
LGTM, looks really good

**Pull request merged**